.pylintrc

```diff
@@ -14,7 +14,7 @@ ignore=CVS
 ignore-patterns=
 
 # Match against paths supporting python regex.
-ignore-paths=^azext_edge/sdk/.*$
```

Comment:
are we going back to the sdk generation path? If so, then please document the versions (node, autorest, autorest-python) and commands used so we don't have to deal with guessing/having new sdks be wildly different.

Reply:
autorest configuration is documented here https://github.com/Azure/azure-iot-ops-cli-extension/wiki/Autorest-config